shared/js/actions.js

```diff
@@ -145,6 +145,14 @@ loop.shared.actions = (function() {
     }),
 
     /**
+     * Notifies that a tile has been received from the other peer.
+     */
+    ReceivedTile: Action.define("receivedTile", {
+      tile: String
+      // sentTimestamp: String (optional)
```

Comment:
How did you choose these? `tile` seems elsewhere to be `@param {Object} tile  The tile to be stored and received by others.` But we probably actually want multiple fields for url and title at least. Are we expecting to use `sentTimestamp` ?